schema/divisions/division.yaml

```diff
@@ -195,4 +195,12 @@ properties:     # JSON Schema: Top-level object properties.
         minItems: 1
         uniqueItems: true
         items: { "$ref": "../defs.yaml#/$defs/propertyDefinitions/id" }
+      capital_of_divisions:
+        description:
+          Division IDs and subtypes of division's this division is capital of. If present,
+          this property will refer to the division IDs of the countries, regions, counties, etc.
```

Comment:
Maybe make a note here the subtype is a direct copy of the division_id's subtype.

Reply:
Could you help me phrase that to make it as clear as possible?

Reply:
"This field indicates when this division is the capital of one or more other divisions. Each item contains the division ID and subtype of the division(s) for which it is the capital".

It is indeed difficult to not end that sentence with "is the capital of".

Reply:
Updated text.